docs/running-on-kubernetes.md

```diff
@@ -1073,6 +1073,13 @@ See the [configuration page](configuration.html) for information on Spark config
   Specify whether executor pods should be deleted in case of failure or normal termination.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.max.executor.failures</code></td>
+  <td>numExecutors * 2, with minimum of 3</td>
```

Comment:
"numExecutors" doesn't really mean anything when you have dynamic allocation on.